docs/diagrams/ModelClassDiagram.puml

```diff
@@ -67,5 +78,4 @@ School -[hidden]right-> Year
 ModelManager-[hidden]- UniqueStudentList
 
 ModelManager -down->"filtered list *" Student
```

Comment:
Additionally, ModelManager also has a "sorted list" association to Student as well